docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.trigger.show.adoc

```diff
@@ -0,0 +1,5 @@
+¦xref::overview/apoc.trigger/apoc.trigger.show.adoc[apoc.trigger.show icon:book[]] +
+
+`CALL apoc.trigger.show(databaseName) | it lists all installed triggers`
+¦label:procedure[]
+¦label:apoc-core[]
```

Comment:
The slight issue I see is that both "apoc.trigger.show" and "apoc.trigger.list" have the same short description "it lists all installed triggers"

I would like to propose
"CALL apoc.trigger.show(databaseName) | it lists all eventually installed triggers for any database"
"CALL apoc.trigger.list() | list all currently installed triggers for all databases for the session's database"

Anything you'd like to suggest is of course welcome!